lbrynet/dht/datastore.py

```diff
@@ -46,7 +66,64 @@ def getPeersForBlob(self, key):
             return [val[0] for val in self._dict[key]]
 
     def removePeer(self, value):
-        for key in self._dict:
+        keys = self.keys()
+        for key in keys:
             self._dict[key] = [val for val in self._dict[key] if val[0] != value]
             if not self._dict[key]:
                 del self._dict[key]
+
+    def Load(self):
+        pass
+
+    def Save(self, contacts):
+        pass
+
+
+class JSONFileDataStore(DictDataStore):
+    def __init__(self, node, path, file_name=None):
+        DictDataStore.__init__(self)
+        self._node = node
+        self._path = os.path.join(path, file_name or "dht_datastore.json")
```

Comment:
Doesn't look like JSONFileDataStore uses self._node
If it does, you have a circular dependency